### ISSUE TITLE
Add a lint for putting a comma/semicolon at the end of a table

### ISSUE THREAD

**Issue by dekkonot** [Kampfkarren/selene#127]
It's good practice to put a comma/semicolon at the end of a table constructor even when there's no more fields. I often forget to do this, so it would be nice to have a lint that reminds me.
```lua
local foo = {
  "bar",
  "baz" -- This should lint here
}
```

Since this lint is stylistic in nature, it should be probably be turned off by default, but I think it would be a nice addition.

**Comment by Kampfkarren** [comment]
I'm still thinking that the best course of action is to only implement style lints if we have fixers.

**Comment by dekkonot** [comment]
I'm not sure what you mean by fixer. Do you mean something to automatically fix the problem? If so, that seems doable, though it might be better to put it in a separate project.

**Comment by Kampfkarren** [comment]
Yeah, since style lints are just a nuisance since if your codebase doesn't comply, it's likely that you'll just have a million lints thrown at you, and you're guaranteed to just turn it off. Having lints off by default goes against the idea that you should have to configure selene as little as possible. 

I'm not sure it would be better as a separate project, it's different from ecosystem to ecosystem. ESLint has fixers built in, while Rust splits Clippy and Rustfmt. If it was a separate project, the codebases would be very similar, and it would probably need its own configuration for things like Roblox specific stuff, which sounds like a pain.

**Comment by Kampfkarren** [comment]
Since this issue was made, StyLua has since come out and proven stability. Is this still something you would want despite that?

**Comment by Dekkonot** [comment]
Ideally, yes, simply because there might be cases where someone runs Selene without StyLua. Though it's less of a priority now, obviously.
